src/Microsoft.AspNetCore.Antiforgery/DefaultAntiforgeryTokenGenerator.cs

```diff
@@ -22,7 +21,8 @@ public class DefaultAntiforgeryTokenGenerator : IAntiforgeryTokenGenerator
             _additionalDataProvider = additionalDataProvider;
         }
 
-        public AntiforgeryToken GenerateCookieToken()
+        /// <inheritdoc />
+        public AntiforgeryToken GenerateCookieToken(HttpContext httpContext)
```

Comment:
> short-sighted and limiting

Hmm, also looks :100:% intentional. See for example the different way an `AntiforgeryToken` is serialized if `IsCookieToken` is `true`. That code "knows" there's no useful user-related information in a cookie token and doesn't attempt to serialize it.